cluster-state-service/Makefile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2016-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Just did a `grep -ilr 'Copyright 2016 Amazon.com' * | xargs -I@ sed -i '' 's/Copyright 2016 Amazon.com/Copyright 2016-2017 Amazon.com/g' @`. The slap-copyright script doesnt replace an existing license. Only adds if it doesnt already exist. So it adds everytime for swagger generate files on make but doesnt update otherwise.